mobile/src/main/res/values/themes.xml

```diff
@@ -17,6 +17,10 @@
         <item name="themedDrawerBackground">?android:colorBackground</item>
         <item name="themedHeaderTextColor">@color/colorAccent_themeDark</item>
 
+        <item name="mal_card_background">@color/about_libraries_window_background</item>
```

Comment:
Going to make a oh theme with dark and one with bright background and use them as parent theme for all oh themes.